src/main/java/br/edu/ifpe/servicon/model/entidades/Endereco.java

```diff
@@ -21,10 +21,142 @@ of this software and associated documentation files (the "Software"), to deal
 SOFTWARE.*/
 package br.edu.ifpe.servicon.model.entidades;
 
+import java.util.Objects;
+
 /**
  *
  * @author Lucas Mendes <lucas.mendes147@live.com>
```

Comment:
@lucasmendsc Este código foi revisado? Está parecendo que não. Se não for, revise-o ou selecione outro DEV para fazê-lo.